ext/appsignal_extension.c

```diff
@@ -635,6 +639,7 @@ void Init_appsignal_extension(void) {
 
   // Event hook installation
   rb_define_singleton_method(Extension, "install_allocation_event_hook", install_allocation_event_hook, 0);
+  rb_define_singleton_method(Extension, "running_in_container", running_in_container, 0);
```

Comment:
Ignore that, you can just call it `running_in_container?` here in the second argument. It already returns a boolean.